features/dd-sdk-android-flags/src/test/kotlin/com/datadog/android/flags/internal/DatadogFlagsClientTest.kt

```diff
@@ -490,18 +496,18 @@ internal class DatadogFlagsClientTest {
 
     // endregion
 
-    // region resolveStructureValue()
```

Comment:
`// region` removal here seems to be wrong, because:

* there is abandoned `// endregion` below
* it is still `resolveStructureValue` test